src/WeakCss.elm

```diff
@@ -73,8 +72,8 @@ namespace name =
     --> class "menu__list"
 
 -}
-addElement : String -> ClassName -> ClassName
```

Comment:
You should update the doc tests

Reply:
yep already done

Reply:
Great, now just increase the version and we are ready to go

Reply:
At least you now know that tests are really running on CI